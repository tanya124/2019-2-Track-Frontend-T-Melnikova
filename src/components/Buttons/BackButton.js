import React from 'react';
import styled from '@emotion/styled';
import back from '../../assets/back.svg';

const Button = styled.img`
	width: 3em;
	margin-left: 10px;
	&:hover {
		filter: invert(0.5);
	}
`;

function BackButton(props) {
	return (
		<Button id="back" src={ back } />
	);
}

export default BackButton;
