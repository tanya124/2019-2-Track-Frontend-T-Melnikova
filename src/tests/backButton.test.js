import React from 'react'
import renderer from 'react-test-renderer'
import BackButton from '../components/Buttons/BackButton'

it('renders correctly', () => {
  const tree = renderer.create(<BackButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
