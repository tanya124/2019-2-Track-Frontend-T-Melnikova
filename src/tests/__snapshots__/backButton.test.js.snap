// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-1v7dot1"
  src="back.svg"
/>
`;
